component/common/LastCake.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box, Container, InputLabel, MenuItem, FormControl, Select } from '@mui/material';
+
+export const LastCake = () => {
+  const [age, setAge] = React.useState('');
```

Comment:
위에서` import {useState } from 'react';` 로 정의하고 아래와 같이 타입 지정하는게 어떄?
```suggestion
  const [age, setAge] = useState<number>();
```